packages/wdio-types/src/Services.ts

```diff
@@ -249,3 +249,10 @@ export interface HookFunctions {
         specs: string[]
     ): void;
 }
+
+export type Service = 'chromedriver' | 'sauce' | 'testingbot' | 'selenium-standalone' |
+    'devtools' | 'applitools' | 'browserstack' | 'appium' | 'firefox-profile' |
+    'crossbrowsertesting' | 'lambdatest' | 'zafira-listener' | 'reportportal' | 'docker' |
+    'wdio-ui5' | 'wiremock' | 'ng-apimock' | 'slack' | 'intercept' | 'docker' |
+    'visual-regression-testing' | 'novus-visual-regression' | 'rerun' | 'winappdriver' |
+    'ywinappdriver' | 'performancetotal' | 'aws-device-farm' | 'ocr-native-apps' | 'wait-for'
```

Comment:
I am not sure if it is a good idea to have such strict type definitions for services given that we have to clearly communicate it when people build their services. I would say for the purpose of this PR this is not necessary.

Reply:
That is only the case when they want to add it to the cli though, is it really a bad thing to have types so that it is correctly added to the CLI? I'd say if anything, it would make it more clear.

Reply:
typing things is generally good but this is not maintainable, the list will get quickly out of sync. Therefor I rather would but `string` as service.